.github/workflows/lint-test-sdk.yml

```diff
@@ -50,17 +51,6 @@ jobs:
   run-tests-with-tox:
     runs-on: ubuntu-latest
     steps:
-      - uses: "actions/checkout@v2"
-      - uses: "actions/setup-python@v2"
-        with: 
-          python-version: '3.9.x'
-      - name: "Install dependencies"
-        run: |
-          python -VV
-          python -m pip install --upgrade pip setuptools wheel
-          python -m pip install -r requirements.txt
-          python -m pip install -r requirements-test.txt
-      - name: 'Set up GCP SDK to download test data'
-        uses: 'google-github-actions/setup-gcloud@v0'
-      - name: "Run tests"
-        run: make test
+      - uses: Eppo-exp/python-sdk/.github/actions/action-test@tp/workflows/remote
```

Comment:
@typotter you seem to forgot removing temp branch reference before merging? ^

This broke PR testing: https://github.com/Eppo-exp/python-sdk/actions/runs/10634064395/job/29480558050?pr=68